src/include/common/vector/value_vector.h

```diff
@@ -35,6 +36,32 @@ class KUZU_API ValueVector {
     DELETE_COPY_AND_MOVE(ValueVector);
     ~ValueVector() = default;
 
+    template<class Func>
+    void forEachNonNull(Func&& func) const {
```

Comment:
Can u open an issue to propagate this to unary, binary & ternary evaluator and assign the issue to @acquamarin